src/components/hero/filters-type/index.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import "./styles.scss";
+
+const FiltersType = ({ categories, selectedCategory, onCategorySelect }) => {
+  return (
+    <div className="filters__type">
+      <div className="categories">
+        {categories.map(category => (
+          <div
+            className={category === selectedCategory ? "categories__item active" : "categories__item"}
```

Comment:
To avoid repetition with `categories__item` you can simply do this.
```suggestion
            className=`categories__item {category === selectedCategory && "active"}`
```